sources/packages/web/src/services/http/EducationProgramOfferingApi.ts

```diff
@@ -0,0 +1,20 @@
+import HttpBaseClient from "./common/HttpBaseClient";
+
+export class EducationProgramOfferingApi extends HttpBaseClient {
+  public async createProgramOffering(
+    locationId: number,
+    programId: number,
+    createProgramOfferingDto: any,
```

Comment:
Please try to make the type explicit here.

Reply:
Done created a contract